code/game/machinery/camera/camera.dm

```diff
@@ -182,34 +181,23 @@
 		interact(user)
 
 	// OTHER
-	else if (can_use() && (istype(I, /obj/item/weapon/paper) || istype(I, /obj/item/device/pda)) && isliving(user))
+	else if (can_use() && (istype(I, /obj/item/weapon/paper) || istype(I, /obj/item/modular_computer)) && isliving(user))
 		var/mob/living/U = user
 		var/obj/item/weapon/paper/X = null
-		var/obj/item/device/pda/P = null
 
 		var/itemname = ""
 		var/info = ""
 		if(istype(I, /obj/item/weapon/paper))
 			X = I
 			itemname = X.name
 			info = X.info
-		else
-			P = I
-			itemname = P.name
-			info = P.notehtml
 		U << "You hold \a [itemname] up to the camera ..."
 		for(var/mob/living/silicon/ai/O in living_mob_list)
 			if(!O.client) continue
 			if(U.name == "Unknown") O << "<b>[U]</b> holds \a [itemname] up to one of your cameras ..."
 			else O << "<b><a href='byond://?src=\ref[O];track2=\ref[O];track=\ref[U];trackname=[U.name]'>[U]</a></b> holds \a [itemname] up to one of your cameras ..."
 			O << browse(text("<HTML><HEAD><TITLE>[]</TITLE></HEAD><BODY><TT>[]</TT></BODY></HTML>", itemname, info), text("window=[]", itemname))
-		for(var/mob/O in player_list)
-			if (istype(O.machine, /obj/machinery/computer/security))
-				var/obj/machinery/computer/security/S = O.machine
-				if (S.current_camera == src)
-					O << "[U] holds \a [itemname] up to one of the cameras ..."
-					O << browse(text("<HTML><HEAD><TITLE>[]</TITLE></HEAD><BODY><TT>[]</TT></BODY></HTML>", itemname, info), text("window=[]", itemname))
-
+			
```

Comment:
This strange block of code was designed to show things to camera viewers, and also to  do a visible message. 
Line 194 should be a visible message call instead, but what about the showing thing?

Reply:
it only showed PDA or paper and messages on them